File: Python Language/Q9.py
# Program to find the maximum number from four numbers

print("Enter four numbers:")
# Getting 4 numbers
n1 = float(input("Number 1: "))
n2 = float(input("Number 2: "))
n3 = float(input("Number 3: "))
n4 = float(input("Number 4: "))

# Using Python's built-in max() function to find the largest
maximum_num = max(n1, n2, n3, n4)

# Printing the result
print(f"The maximum number is: {maximum_num}")

File: C Language/Q9.c
#include <stdio.h>

int main() {
    // Declaring four numbers
    int n1 = 10, n2 = 55, n3 = 30, n4 = 45;
    int max;

    // Assuming first is max, then comparing with others
    max = n1;
    
    if (n2 > max) {
        max = n2;
    }
    if (n3 > max) {
        max = n3;
    }
    if (n4 > max) {
        max = n4;
    }

    printf("The maximum number is: %d\n", max);
    return 0;
}